src/k8s-engine/homestead.js

```diff
@@ -55,18 +55,17 @@ async function ensureHelmChart(state) {
   }
 
   switch (state) {
-    case State.NONE: {
+    case State.NONE:
```

Comment:
To actually finish #38 we'll need to add a `State.FULL_RANCHER` or whatever we decide to call it.  This is putting the cart before the horse a bit, but since it's already clear we're doing it…

Reply:
cool